microservices/authorization/migrations/permissions/list/methods/payment-stripe.json

```diff
@@ -76,7 +76,7 @@
     "method": "customer.create",
     "description": "Create new customer",
     "allowGroup": [
-      "admin"
+      "user"
```

Comment:
why this method without protection?

Reply:
reverted. I'll add protection in the bloom api